scripts/lib/mkcloud-common.sh

```diff
@@ -2,7 +2,8 @@
 
 # defaults for generic common variables
 : ${admin_image_password:='linux'}
-: ${clouddata:=$(dig -t A +short clouddata.nue.suse.com)}
+: ${clouddatadns:=clouddata.nue.suse.com}
+: ${clouddata:=$(dig -t A +short $clouddatadns)}
```

Comment:
please comment that clouddata must be an IP, it was required before already, but now as we change the line anyway ...